### PR TITLE
Updating test case descriptions for `audio` or `video` has no audio that plays automatically' rules

### PR DESCRIPTION
**Pull request by daniel-montalvo**
Updating test case descriptions for `audio` or `video` has no audio that plays automatically' rules
Need for Final Call:
This will require a 1 week Final Call.  >>

---

### **When creating PR:**

- [x] Make sure you're requesting to **pull a branch** (right side) to the `develop` branch (left side).

### **After creating PR:**

- [x] Add yourself (and co-authors) as "Assignees" for PR.
- [x] Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- [ ] Close the issue that the PR resolves (and make sure the issue is referenced in the top of this comment)
- [x] Optionally request feedback from anyone in particular by assigning them as "Reviewers".

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Final Call period. In case of disagreement, the longer period wins.

### CODE REVIEW

File: _rules/auto-play-audio-does-not-exceed-3-seconds-aaa1bf.md [hunk 1]
```diff
@@ -62,15 +62,15 @@ There are no major accessibility support issues known for this rule.
 
 #### Passed Example 1
 
-The `<audio>` element does not play automatically for more than 3 seconds.
+This `<audio>` element does not play automatically for more than 3 seconds.
```
Comment (jkodu) [inline _rules/auto-play-audio-does-not-exceed-3-seconds-aaa1bf.md]:
```suggestion
This `audio` element does not play automatically for more than 3 seconds.
```

File: _rules/auto-play-audio-does-not-exceed-3-seconds-aaa1bf.md [hunk 2]
```diff
@@ -62,15 +62,15 @@ There are no major accessibility support issues known for this rule.
...
 
 ```html
 <audio src="/test-assets/moon-audio/moon-speech.mp3#t=25" autoplay></audio>
 ```
 
 #### Passed Example 2
 
-The `<video>` element audio output does not last longer than 3 seconds.
+The `audio` output of this `<video> element does not last longer than 3 seconds.
```
Comment (jkodu) [inline _rules/auto-play-audio-does-not-exceed-3-seconds-aaa1bf.md]:
```suggestion
This `video` element's `audio` output does not last longer than 3 seconds.
```

File: _rules/auto-play-audio-does-not-exceed-3-seconds-aaa1bf.md [hunk 3]
```diff
@@ -104,7 +104,7 @@ The `video` element automatically plays some audio for more than 3 seconds.
 
 #### Inapplicable Example 1
 
-The `<video>` element audio autoplays for longer than 3 seconds, but is `muted`.
+The audio of this `<video>` element autoplays for longer than 3 seconds but is `muted`.
```
Comment (jkodu) [inline _rules/auto-play-audio-does-not-exceed-3-seconds-aaa1bf.md]:
```suggestion
This `video` element has audio that autoplays for longer than 3 seconds but is muted.
```

File: _rules/auto-play-audio-does-not-exceed-3-seconds-aaa1bf.md [hunk 4]
```diff
@@ -115,7 +115,7 @@ The `<video>` element audio autoplays for longer than 3 seconds, but is `muted`.
 
 #### Inapplicable Example 2
 
-The `video` element `src` file has no audio output.
+The `src` file of this `video` element has no audio output.
```
Comment (jkodu) [inline _rules/auto-play-audio-does-not-exceed-3-seconds-aaa1bf.md]:
```suggestion
This `video` element refers to a source file that has no audio output.
```

File: _rules/auto-play-audio-has-control-mechanism-4c31df.md [hunk 5]
```diff
@@ -176,7 +176,7 @@ The `<video>` has a [mechanism](https://www.w3.org/TR/WCAG21/#dfn-mechanism) to
 
 #### Failed Example 4
 
-The `<video>` has a [mechanism](https://www.w3.org/TR/WCAG21/#dfn-mechanism) to pause or stop or turn the audio volume off, but the control `<button>` elements do not have accessible names.
+This `video` element has a [mechanism](https://www.w3.org/TR/WCAG21/#dfn-mechanism) to pause, stop, or turn the audio volume off but its `<button>` elements do not have accessible names.
```
Comment (WilcoFiers) [inline _rules/auto-play-audio-has-control-mechanism-4c31df.md:179]:
```suggestion
This `video` element has a [mechanism](https://www.w3.org/TR/WCAG21/#dfn-mechanism) to pause, stop, or turn the audio volume off but its `button` elements do not have accessible names.
```

File: _rules/auto-play-audio-has-control-mechanism-4c31df.md [hunk 6]
```diff
@@ -211,7 +211,7 @@ The `<video>` has a [mechanism](https://www.w3.org/TR/WCAG21/#dfn-mechanism) to
 
 #### Failed Example 5
 
-The `<video>` has a [mechanism](https://www.w3.org/TR/WCAG21/#dfn-mechanism) to pause or stop or turn the audio volume off, but the mechanism is not included in the accessibility tree.
+This `video` element has a [mechanism](https://www.w3.org/TR/WCAG21/#dfn-mechanism) to paus, stop, or turn the audio volume off but the mechanism is not included in the accessibility tree.
```
Comment (WilcoFiers) [inline _rules/auto-play-audio-has-control-mechanism-4c31df.md:214]:
```suggestion
This `video` element has a [mechanism](https://www.w3.org/TR/WCAG21/#dfn-mechanism) to pause, stop, or turn the audio volume off but the mechanism is not included in the accessibility tree.
```

Comment (daniel-montalvo) [reply _rules/auto-play-audio-has-control-mechanism-4c31df.md:214]: I don't understand what's changed in this suggestion. Will commit the change but I would appreciate if you could point to the specifics.

Comment (Jym77) [reply _rules/auto-play-audio-has-control-mechanism-4c31df.md]: There was a missing 'e' at the end of "paus"/"pause".
(you accidentally removed it when changing "pause or stop or turn" to "pause, stop, or turn")

File: _rules/auto-play-audio-does-not-exceed-3-seconds-aaa1bf.md [hunk 7]
```diff
@@ -62,15 +62,15 @@ There are no major accessibility support issues known for this rule.
...
+This `audio` element does not play automatically for more than 3 seconds.
 
 ```html
 <audio src="/test-assets/moon-audio/moon-speech.mp3#t=25" autoplay></audio>
 ```
 
 #### Passed Example 2
 
-The `<video>` element audio output does not last longer than 3 seconds.
+This `video` element's `audio` output does not last longer than 3 seconds.
```
Comment (carlosapaduarte) [inline _rules/auto-play-audio-does-not-exceed-3-seconds-aaa1bf.md:73]: The audio in this sentence does not refer to an element or attribute so it should not be between backticks.

```suggestion
This `video` element's audio output does not last longer than 3 seconds.
```

**Pull request merged**